robot/Cumulus/tests/browser/gift_entry/reorder_template.robot

```diff
@@ -0,0 +1,79 @@
+*** Settings ***
+
+Resource        robot/Cumulus/resources/NPSP.robot
+Library         cumulusci.robotframework.PageObjects
+...             robot/Cumulus/resources/NPSPSettingsPageObject.py
+...             robot/Cumulus/resources/GiftEntryPageObject.py
+Suite Setup     Run keywords
+...             Open Test Browser
+...             API Check And Enable Gift Entry
+Suite Teardown  Capture Screenshot and Delete Records and Close Browser
+
+*** Keywords ***
+
+Get Template Builder Field Names
+  @{builder_form_fields} =  Get Webelements  xpath://lightning-input[@class="slds-truncate slds-form-element"]//label
```

Comment:
Good practice to move the element locator in the locators file for later reference or to make the refactoring easier.
Please move the locator to locators file. Please also see if there is a similar element that already exists to make use of it.

Reply:
Moved this to a keyword.